--- Gtk2-Spell-1.03.orig/Makefile.PL	2003-09-22 02:45:59.000000000 +0200
+++ Gtk2-Spell-1.03/Makefile.PL	2010-05-12 08:31:44.000000000 +0200
@@ -11,8 +11,8 @@
 our %build_reqs = (
 	'perl-ExtUtils-Depends'   => '0.1',
 	'perl-ExtUtils-PkgConfig' => '0.1',
-	'perl-Glib'               => '1.00rc2',
-	'perl-Gtk2'               => '1.00rc2',
+	'perl-Glib'               => '1.200',
+	'perl-Gtk2'               => '1.200',
 	'GtkSpell'                => '2.0.0',
 );
 
